generators/app/index.js

```diff
@@ -95,6 +97,34 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    askForGithubAccount: function () {
+      var done = this.async();
+
+      githubUsername(this.user.git.email(), function (err, username) {
+        if (err) {
+          this.log.error(err);
```

Comment:
if we have an error we want to raise an exception and exit the process. Unless we have a way of handling it.

Right now this is wrong as the code will continue to execute.